contrib/mmap/mmap_extract.py

```diff
@@ -115,7 +115,7 @@ def get_subprocess_config() -> SubprocessConfig:
 def run_move_map_gen(map_id: int) -> None:
     config = get_subprocess_config()
     subprocess.check_call(
-        (config.executable, str(map_id), "--silent", *sys.argv[1:]),
+        (config.executable, str(map_id), "--silent", "--threads", "1", *sys.argv[3:]),
```

Comment:
`*sys.argv[1:]` has to stay as it is; this has nothing to do with the added `--threads 1` (it just passes along the arguments that the Python script was invoked with).